rules/framework.bzl

```diff
@@ -340,20 +355,87 @@ def _apple_framework_packaging_impl(ctx):
             swift_common.create_module(name = swiftmodule_name, swift = swift_module),
         ]
 
-    # Eventually we need to remove any reference to objc provider
-    # and use CcInfo instead, see this issue for more details: https://github.com/bazelbuild/bazel/issues/10674
-    objc_provider = apple_common.new_objc_provider(**objc_provider_fields)
+    direct_headers = []
+
+    if virtualize_frameworks:
+        import_vfsoverlays = []
+        for dep in ctx.attr.vfs:
+            if not VFSOverlayInfo in dep:
+                continue
+            import_vfsoverlays.append(dep[VFSOverlayInfo].vfs_info)
+
+        # Progated interface headers - this must encompass all of them
+        propagated_interface_headers = []
```

Comment:
this is an array of depset correct?

Reply:
Yep, you'll find where it's created downwind of this line https://github.com/bazel-ios/rules_ios/pull/277/files/2c7b4ba65c40c684f4f8e7efbed35c61e84ca582#diff-b5ab983bf150dfa86b7e361224e1c9b0b9856b0b57b5893dbc088c6ca6d8ba52R397